templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
```

Comment:
We need to add to this comment that we're also possibly displaying one-time opportunities here as well.